lib/mysql.js

```diff
@@ -465,72 +466,76 @@ MySQL.prototype._buildWhere = function (model, conds) {
       }
       // The value is not an array, fall back to regular fields
     }
-    var keyEscaped = self.columnEscaped(model, key);
-    var val = self.toDatabase(props[key], conds[key]);
-    if (conds[key] === null || conds[key] === undefined) {
-      cs.push(keyEscaped + ' IS NULL');
-    } else if (conds[key] && conds[key].constructor.name === 'Object') {
-      var condType = Object.keys(conds[key])[0];
-      var sqlCond = keyEscaped;
-      if ((condType === 'inq' || condType === 'nin') && val.length === 0) {
-        cs.push(condType === 'inq' ? 0 : 1);
-        return true;
-      }
-      switch (condType) {
-        case 'gt':
-          sqlCond += ' > ';
-          break;
-        case 'gte':
-          sqlCond += ' >= ';
-          break;
-        case 'lt':
-          sqlCond += ' < ';
-          break;
-        case 'lte':
-          sqlCond += ' <= ';
-          break;
-        case 'between':
-          sqlCond += ' BETWEEN ';
-          break;
-        case 'inq':
-          sqlCond += ' IN ';
-          break;
-        case 'nin':
-          sqlCond += ' NOT IN ';
-          break;
-        case 'neq':
-          sqlCond += ' != ';
-          break;
-        case 'like':
-          sqlCond += ' LIKE ';
-          break;
-        case 'nlike':
-          sqlCond += ' NOT LIKE ';
-          break;
-      }
-      sqlCond += (condType === 'inq' || condType === 'nin') ? '(' + val + ')' : val;
-      cs.push(sqlCond);
+    if (key === '$where') {
+      cs.push(conds[key]);
```

Comment:
I consider about this.At present,I have 2 way.
1.In `$where` to use `$` or `this` prefix.
```js
Model.find({
  filter:{
    $where:'this.star>this.watch'//or :$star>$watch
  }
})
```
2.Or we can change the value.
```js
Model.find({
  filter:{
    star:{
      gt:'this.watch'//or :$watch
    }
  }
})
```
Which way you prefer~?